test/src/main/java/org/apache/accumulo/test/functional/TabletAvailabilityIT.java

```diff
@@ -43,12 +43,23 @@
 import org.apache.accumulo.core.client.admin.TabletAvailability;
 import org.apache.accumulo.core.data.Mutation;
 import org.apache.accumulo.core.data.Range;
+import org.apache.accumulo.core.metadata.AccumuloTable;
 import org.apache.accumulo.harness.AccumuloClusterHarness;
 import org.apache.hadoop.io.Text;
 import org.junit.jupiter.api.Test;
 
 public class TabletAvailabilityIT extends AccumuloClusterHarness {
 
+  @Test
+  public void testSystemFails() throws Exception {
+    try (AccumuloClient client = Accumulo.newClient().from(getClientProps()).build()) {
+      for (AccumuloTable t : AccumuloTable.values()) {
+        assertThrows(IllegalArgumentException.class, () -> client.tableOperations()
+            .setTabletAvailability(t.tableName(), new Range(), HOSTED));
+      }
+    }
+  }
```

Comment:
Could try to set the system table to UNHOSTED and ensure that it throws, then get the tablet availability after and ensure it's still HOSTED

Reply:
Doh! I meant for that to be UNHOSTED. Good catch.